plot_friends.py

```diff
@@ -86,5 +69,13 @@ def friends():
     plt.legend(loc='upper left', ncol=2)
     plt.show()
 
+    print('\nSome statistics based on the data: ', maxdays)
```

Comment:
Please make it explicit that this is the number of days being printed
OR
just don't print maxdays

Reply:
ooh, I didn't mean to print it there, it was just for testing purpose. Forgot to delete it :sweat_smile: